.github/workflows/website-new-version.yml

```diff
@@ -21,7 +20,7 @@ jobs:
           fetch-tags: true
       - uses: actions/checkout@v4
         with:
-          repository: 'Openlineage/openlineage-site'
+          repository: 'OpenLineage/openlineage-site'
```

Comment:
Should the monorepo be here instead?

Reply:
No, it needs `openlineage-site` to create new version of docs.